### PR TITLE
workshop description

### PR DESCRIPTION
**Pull request by jburel**
add brief workshop description

cc @will-moore @pwalczysko

### CODE REVIEW

**Review by sbesson — CHANGES_REQUESTED** [review]
As mentioned, more description is better than less.
Main stylistic comment is that this PR uses a mixture of "This session will" vs "We will". My vote goes towards unifying using the first format as this is closer to what has been used for the other Events on the programme.

File: _data/ome2021.json [hunk 1]
```diff
@@ -584,6 +586,7 @@
             {
               "@type": "Event",
               "name": "IDR Workshop",
+              "description": "This session will first explore metadata associated to IDR data using the Web Interface. Then, we will demonstrate how to retrieve data and metadata using the APIs.",
```
Comment (pwalczysko) [inline _data/ome2021.json:589]: Introduction to IDR ecosystem will be shown on concrete examples of IDR studies. Images and metadata mining and comparison workflows will be presented. This workshop is going to show both to those who want to script as well as to those who do not how to get value from IDR.  The first, hands-on section will show the cross-studies searches using OMERO.web. The second section will expand on these possiblities showing complex metadata mining and plotting as well as image reanalysis in Jupyter notebooks.

File: _data/ome2021.json [hunk 2]
```diff
@@ -684,6 +687,7 @@
...
```
Comment (pwalczysko) [inline _data/ome2021.json:690]: Introduction to IDR ecosystem will be shown on concrete examples of IDR studies. Images and metadata mining and comparison workflows will be presented. This workshop is going to show both to those who want to script as well as to those who do not how to get value from IDR.  The first, hands-on section will show the cross-studies searches using OMERO.web. The second section will expand on these possiblities showing complex metadata mining and plotting as well as image reanalysis in Jupyter notebooks.

File: _data/ome2021.json (hunk 1 above)
Comment (pwalczysko) [reply _data/ome2021.json:589]: Trying to address @sbesson 's encouragement for longer description.

File: _data/ome2021.json [hunk 3]
```diff
@@ -311,7 +311,8 @@
             },
             {
               "@type": "Event",
-              "name": "OMERO.figure workshop",
+              "name": "OMERO.figure Workshop",
+              "description": "This session will show how figures can quickly be created using OMERO.figure.",
```
Comment (pwalczysko) [inline _data/ome2021.json:315]: I would add:

"New workflows showing a more advanced work with ROIs in OMERO.figure will be presented."

File: _data/ome2021.json [hunk 4]
```diff
@@ -955,6 +960,7 @@
             {
               "@type": "Event",
               "name": "OMERO trainers Workshop",
+              "description": "This session will introduce the materials available for running a successful OME workshop and discuss possible setups.",
```
Comment (pwalczysko) [inline _data/ome2021.json:963]:
```suggestion
              "description": "This session will introduce the materials available for running a successful OME workshop, lead you through steps necessary for replicating an OME Team training setups and discuss possible imrpovements and modifications of such setups.",
```

File: _data/ome2021.json (hunk 2 above)
Comment (jburel) [reply _data/ome2021.json:690]: This might be too much on the page.

File: _data/ome2021.json [hunk 5]
```diff
@@ -866,6 +870,7 @@
...
```
Comment (pwalczysko) [inline _data/ome2021.json:873]:
```suggestion
              "description": " "description": "This session will introduce the materials available for running a successful OME workshop, lead you through steps necessary for replicating an OME Team training setups and discuss possible imrpovements and modifications of such setups.",
```

File: _data/ome2021.json (hunk 2 above)
Comment (jburel) [reply _data/ome2021.json]: Used some of the text. I removed ``This workshop is going to show both to those who want to script as well as to those who do not how to get value from IDR.``

File: _data/ome2021.json [hunk 6]
```diff
@@ -866,6 +870,7 @@
             {
               "@type": "Event",
               "name": "OMERO trainers Workshop",
+              "description": "This session will introduce the materials available for running a successful OME workshop, lead you through steps necessary for replicating an OME Team training setups and discuss possible imrpovements and modifications of such setups.",
```
Comment (pwalczysko) [inline _data/ome2021.json:873]:
```suggestion
              "description": "This session will introduce the materials available for running a successful OME workshop, lead you through the steps necessary for replicating OME Team training setups and discuss possible imrpovements and modifications of such setups.",
```

File: _data/ome2021.json [hunk 7]
```diff
@@ -955,6 +960,7 @@
...
```
Comment (pwalczysko) [inline _data/ome2021.json:963]:
```suggestion
              "description": "This session will introduce the materials available for running a successful OME workshop, lead you through the steps necessary for replicating OME Team training setups and discuss possible imrpovements and modifications of such setups.",
```

**Comment by pwalczysko** [comment]
fixed two typos

**Comment by sbesson** [comment]
Is this deployed on the staging website for review?

**Comment by jburel** [comment]
It should be see for example https://snoopycrimecop.github.io/www.openmicroscopy.org/events/ome-community-meeting-2021/day3/

**Review by sbesson — APPROVED** [review]

**Pull request merged**